Bottom: 56158c2a6689a6e842cd419aa9efbd01ed9343ab
Top:    2a24675b11f4ee9ea4c4291977f321cd48b2b18c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 17:54:43 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
